src/components/pickers/mixins/date-table.js

```diff
@@ -53,7 +53,7 @@ export default {
       const day = new Date(
         this.tableYear,
         this.tableMonth
-      ).getDay() - this.days.indexOf(this.firstDayOfWeek)
+      ).getDay() - this.firstDayOfWeek
```

Comment:
since we are accepting firstDayOfWeek as a string, just do parseInt(this.firstDayOfWeek)